tests/timeseries/test_moving_window.py

```diff
@@ -133,3 +143,6 @@ async def test_resampling_window(fake_time: time_machine.Coordinates) -> None:
     assert len(window) == window_size / output_sampling
     for value in window:  # type: ignore
         assert 4.9 < value < 5.1
+
+
+asyncio.run(test_access_window_by_int_slice())
```

Comment:
Why this? `pytest` should properly run `async` tests in a async context automatically.

```suggestion
```

Reply:
removed. good catch!